### PR TITLE
Remove pixels on non-event and add onBidWon

### PR DESCRIPTION
**Pull request by SublimeLeo**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Changes: 
- Removed pixels
- Added onBidWon event pixel

Test parameters for validating bids
```
{
  bidder: 'sublime',
  params: {
    zoneId: 1
  }
}
```

### CODE REVIEW

**Comment by SublimeJeremy** [comment]
@SublimeLeo C'est bien rebase depuis le master de https://github.com/prebid/Prebid.js ?

**Comment by SublimeLeo** [comment]
@SublimeJeremy Maintenant oui!

File: modules/sublimeBidAdapter.js [hunk 1]
```diff
@@ -187,30 +180,38 @@ function interpretResponse(serverResponse, bidRequest) {
...
   return bidResponses;
 }
 
+/**
+ * Send pixel when bidWon event is triggered
+ * @param {Object} timeoutData
+ */
+function onBidWon(bid) {
+  log('Bid won', bid);
+  sendEvent('bidwon', true);
```
Comment (fgcloutier) [inline modules/sublimeBidAdapter.js]: Ne correspond pas à la signature de sendEvent.

```suggestion
  sendEvent('bidwon');
```

File: modules/sublimeBidAdapter.js [hunk 2]
```diff
@@ -187,30 +180,38 @@ function interpretResponse(serverResponse, bidRequest) {
...
+}
+
 /**
  * Send debug when we timeout
  * @param {Object} timeoutData
  */
 function onTimeout(timeoutData) {
   log('Timeout from adapter', timeoutData);
-  sendEvent('dbidtimeout', true);
+  sendEvent('bidtimeout', true);
```
Comment (fgcloutier) [inline modules/sublimeBidAdapter.js]: Idem.

```suggestion
  sendEvent('bidtimeout');
```

**Review by ghourlier — APPROVED** [review]

File: modules/sublimeBidAdapter.js [hunk 3]
```diff
@@ -38,10 +39,8 @@ export function setState(value) {
 /**
  * Send pixel to our debug endpoint
  * @param {string} eventName - Event name that will be send in the e= query string
- * @param {Boolean=} isMandatoryPixel - If set to true, will always send the pixel
  */
-export function sendEvent(eventName, isMandatoryPixel = false) {
-  const shoudSendPixel = (isMandatoryPixel || state.debug);
+export function sendEvent(eventName) {
```
Comment (ghourlier) [inline modules/sublimeBidAdapter.js:43]: tu utilise un deuxieme parametre à l'apelle pourquoi ?

**Review by ghourlier — CHANGES_REQUESTED** [review]

**Review by fgcloutier — APPROVED** [review]

**Review by SublimeJeremy — APPROVED** [review]

**Pull request merged**